R/zzz.R

```diff
@@ -13,46 +13,41 @@
 
 #' Set package options with regard to computation times
 #' 
-#' These functions set options of the bfast and strucchangeRcpp package to enable
-#' faster computations. The fast options should return equal results but
-#' require the \code{strucchangeRcpp} package, which includes
-#' matrix-based functions. Notice that only some functions of the \code{bfast}
-#' package make use of these options.
+
+#' These functions set options of the bfast and strucchangeRcpp packages to enable
+#' faster computations. By default (\code{set_default_options}), these optimizations are 
+#' enabled. Notice that only some functions of the \code{bfast}
+#' package make use of these options. \code{set_fastt_options} is an alias for \code{set_default_options}.
```

Comment:
Misspelled option name:
```suggestion
#' package make use of these options. \code{set_fast_options} is an alias for \code{set_default_options}.
```